docker-images/openemr/run_openemr.sh

```diff
@@ -1,9 +1,53 @@
 #!/bin/sh
+# to be run by root with OpenEMR root dir as the CWD
 
 # allows customization of openemr credentials, preventing the need for manual setup
 #  - Setting db parameters via MYSQL_HOST, MYSQL_USER, MYSQL_PASS
 #  - Setting first-time account creation via OE_USER, OE_PASS
 #  - All of these must be passed to automate the first-time setup process
+set -e
+
+auto_setup() {
+    chmod -R 600 .
+    php auto_configure.php -f \
+        iuser=$OE_USER \
+        iuserpass=$OE_PASS \
+        login=$MYSQL_USER \
+        rootpass=$MYSQL_ROOT_PASS \
+        pass=$MYSQL_PASS \
+        server=$MYSQL_HOST \
+
+    #do I need to activate mod_rewrite?
```

Comment:
As I recall, you dealt with this in the apache conf script (note you can testing by ensuring just don't get error screen when go to the Modules menu item in OpenEMR).

Reply:
Oh whoops. That line was actually my personal note and one I meant to remove. I'll take that out when I clean this up in the morning.